ocpp/v201/enums.py

```diff
@@ -1350,3 +1351,1347 @@ class SecurityEventType(str, Enum):
+    tries_max_limit = "Tries(MaxLimit)"
+
+
+class ConnectorProtectionReleaseVariableName(str, Enum):
+    """
+    Variable names where the component type is ConnectorProtectionRelease
+    See PhysicalComponentName for referenced physical component
+    """
+
+    enabled = "Enabled"
+    active = "Active"
+    problem = "Problem"
+    tripped = "Tripped"
+
+
+class ControllerVariableName(str, Enum):
+    """
+    Variable names where the component type is Controller
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    ec_variant = "ECVariant"
+    firmware_version = "FirmwareVersion"
+    interval_heartbeat = "Interval[Heartbeat]"
+    manufacturer = "Manufacturer"
+    max_msg_elements = "MaxMsgElements"
+    model = "Model"
+    problem = "Problem"
+    selftest_active = "SelftestActive"
+    selftest_active_set = "SelftestActive(Set)"
+    serial_number = "SerialNumber"
+    version_date = "VersionDate"
+    version_number = "VersionNumber"
+
+
+class ControlMeteringVariableName(str, Enum):
+    """
+    Variable names where the component type is ControlMetering
+    See PhysicalComponentName for referenced physical component
+    """
+
+    power = "Power"
+    ac_current = "ACCurrent"
+    dc_current = "DCCurrent"
+    dc_voltage = "DCVoltage"
+
+
+class CPPWMControllerVariableName(str, Enum):
+    """
+    Variable names where the component type is CPPWMController
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    dc_voltage = "DCVoltage"
+    enabled = "Enabled"
+    percent = "Percent"
+    problem = "Problem"
+    selftest_active = "SelftestActive"
+    selftest_active_set = "SelftestActive(Set)"
+    state = "State"
+
+
+class DataLinkVariableName(str, Enum):
+    """
+    Variable names where the component type is DataLink
+    See PhysicalComponentName for referenced physical component
+    """
+
+    active = "Active"
+    complete = "Complete"
+    enabled = "Enabled"
+    fallback = "Fallback"
+    iccid = "ICCID"
+    imsi = "IMSI"
+    network_address = "NetworkAddress"
+    problem = "Problem"
+    signal_strength = "SignalStrength"
+
+
+class DisplayVariableName(str, Enum):
+    """
+    Variable names where the component type is Display
+    See PhysicalComponentName for referenced physical component
+    """
+
+    color = "Color"
+    count_height_in_chars = "Count[HeightInChars]"
+    count_width_in_chars = "Count[WidthInChars]"
+    data_text_visible = "DataText[Visible]"
+    enabled = "Enabled"
+    problem = "Problem"
+    state = "State"
+
+
+class DistributionPanelVariableName(str, Enum):
+    """
+    Variable names where the component type is DistributionPanel
+    See PhysicalComponentName for referenced physical component
+    """
+
+    charging_station = "ChargingStation"
+    charging_station_identities = "ChargingStationIdentities"
+    charging_station_index = "ChargingStationIndex"
```

Comment:
I can't find these entries, neither in OCPP-2.0.1_part2_appendices_v13.pdf 3.2.24. DistributionPanel (or the rest of the PDF) nor in the CSVs or the XSLX. Did I miss them? Where do they come from?

Reply:
Corrected